library/src/main/java/in/krharsh17/barview/Constants.java

```diff
@@ -1,7 +1,6 @@
 package in.krharsh17.barview;
 
 public interface Constants {
-
```

Comment:
This change wasn't needed. However, I'll not delay this PR any more. But please take care from the next time.

Reply:
Ill do that @krharsh17